tests/action.test.ts

```diff
@@ -360,7 +360,7 @@ describe('Integration tests', () => {
     } catch (error) {
       expect(error).toMatchInlineSnapshot(`
         {
-          "message": "Testing Farm test failed - No summary provided",
+          "message": "Build finished - Infra problems - No summary provided",
```

Comment:
Could we use an empty string instead of "No summary provided" if summary is not available?

Reply:
Also, I think the case, when there are Infra Problems, is really the one when the build did not finish - was interrupted by the infra problems. So in this one case I would rather used "Failed" instead of "Build Finished."

Reply:
@zmiklank , addressed in 80ecaf2